frontend/model/contracts/chatroom.js

```diff
@@ -350,20 +353,20 @@ sbp('chelonia/defineContract', {
 
         const me = sbp('state/vuex/state').loggedIn.username
 
-        if (me === meta.username) {
+        if (me === meta.username && data.type !== MESSAGE_TYPES.INTERACTIVE) {
```

Comment:
<img src='https://user-images.githubusercontent.com/17641213/236653920-7511dc56-050b-4b3a-87eb-61f553f11fb2.png' width='440'>

executing `messageReceivePostEffect` (sending browser&sound notifications for the message, activating the chatroom badge for 'unread' state) for above type message (`MESSAGE_TYPES.INTERACTIVE`) should be done regardless of which user it was triggered by.